### PR TITLE
feat(design-system): modals updated!

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: apps/root/src/common/components/contact-modal/edit-contact-modal/index.tsx [hunk 1]
```diff
@@ -7,13 +7,17 @@ import useStoredLabels from '@hooks/useStoredLabels';
...
+import styled from 'styled-components';
 
 interface AddContactModalProps {
   contact: Contact;
   setActiveModal: SetStateCallback<ContactListActiveModal>;
+  goBack?: () => void;
 }
 
-const EditContactModal = ({ contact, setActiveModal }: AddContactModalProps) => {
+const StyledInputsContainer = styled(ContainerBox)``;
```
Comment (th0rOdinson) [inline apps/root/src/common/components/contact-modal/edit-contact-modal/index.tsx:18]: empty style gone gone :D

File: apps/root/src/common/components/contact-modal/edit-contact-modal/index.tsx [hunk 2]
```diff
@@ -77,15 +81,22 @@ const EditContactModal = ({ contact, setActiveModal }: AddContactModalProps) =>
...
+        {goBack && (
+          <Button variant="outlined" size="large" onClick={goBack} fullWidth>
+            <FormattedMessage description="add-contact-modal.buttons.cancel" defaultMessage="Cancel" />
+          </Button>
         )}
-      </Button>
-    </ContainerBox>
+        <Button variant="contained" size="large" onClick={onEditContact} fullWidth disabled={isLoading}>
+          {isLoading ? (
+            <CenteredLoadingIndicator size={32} />
```
Comment (th0rOdinson) [inline apps/root/src/common/components/contact-modal/edit-contact-modal/index.tsx:93]: We have an inconsistency in our user experience here. As we discussed some time ago, Contact Modal should not have a loading state, it should be closed and then the notification will confirm de operation status (I think this is done for creation, but awaited for loading)

File: apps/root/src/common/components/modify-settings-modal/index.tsx [hunk 3]
```diff
@@ -720,10 +720,7 @@ const ModifySettingsModal = ({ position, open, onCancel }: ModifySettingsModalPr
       actions={actions}
       actionsAlignment="horizontal"
     >
-      <Grid container direction="column" alignItems="stretch" spacing={6}>
-        <Grid item xs={12}>
-          <DividerBorder2 />
-        </Grid>
+      <Grid container direction="column" alignItems="stretch" spacing={5}>
```
Comment (th0rOdinson) [inline apps/root/src/common/components/modify-settings-modal/index.tsx:723]: spacing of 20 px is deprecated in the new design system, may be 16 or 24 px

File: apps/root/src/pages/transfer/components/confirm-transfer-modal/index.tsx [hunk 4]
```diff
@@ -193,93 +177,110 @@ const ConfirmTransferModal = ({
...
+        },
+        {
+          label: <FormattedMessage description="transfer transferButton" defaultMessage="Transfer" />,
+          color: 'primary',
+          variant: 'contained',
+          onClick: onTransfer,
+        },
+      ]}
+    >
+      <ContainerBox flexDirection="column">
```
Comment (th0rOdinson) [inline apps/root/src/pages/transfer/components/confirm-transfer-modal/index.tsx:208]: :O 
Screenshot 2024-10-03 at 2 08 44 PM

**Review by th0rOdinson — COMMENTED** [review]
Nice looking modals here!

File: apps/root/src/common/components/contact-modal/edit-contact-modal/index.tsx (hunk 2 above)
Comment (FiboApe) [reply apps/root/src/common/components/contact-modal/edit-contact-modal/index.tsx:93]: I get that, I don't think its something that we should tackle now, we can take it up with the ux designer

**Review by th0rOdinson — APPROVED** [review]

**Pull request merged**